rectools/models/__init__.py

```diff
@@ -35,12 +35,16 @@
 `models.PopularInCategoryModel`
 `models.PureSVDModel`
 `models.RandomModel`
+`models.nn.bert4rec.BERT4RecModel`
+`models.nn.sasrec.SASRecModel`
 """
 
 from .ease import EASEModel
 from .implicit_als import ImplicitALSWrapperModel
 from .implicit_bpr import ImplicitBPRWrapperModel
 from .implicit_knn import ImplicitItemKNNWrapperModel
+from .nn.bert4rec import BERT4RecModel
+from .nn.sasrec import SASRecModel
```

Comment:
Shouldn't be these rows wrapped with try..except?

Reply:
Thank you. Fixed